common/types/organization.go

```diff
@@ -15,7 +15,25 @@ type EditOrgReq struct {
 	Description string `json:"description" example:"org description"`
 	//TODO:rename json field name to 'name", need to negotiate with Portal engineer
 	//Org unique identifier
-	Name string `json:"path" example:"org_name_1"`
+	Name        string `json:"path" example:"org_name_1"`
+	CurrentUser string `json:"current_user"`
+}
+
+// This struct only used in swagger API
+type EditOrgBody struct {
```

Comment:
why this struct?

Reply:
The swagger API of edit organization does not need parameter `name`, the `name` is already specified in the url. So I defined a new struct to represent the required fields in the API body.

Reply:
This problem is solved by using `json:"-"`.